curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/66bbb13f90639e7a6d664153.md

```diff
@@ -0,0 +1,169 @@
+---
+id: 66bbb13f90639e7a6d664153
+title: Step 34
+challengeType: 0
+dashedName: step-34
+---
+
+# --description--
+
+While `getMode` is looking great, the `forEach` loop can simplified further. In order to do so, the function will be turned into an `arrow function expression`. This kind of expression is known for not using curly braces and shorter syntax. 
+
+An example of such a expression is the following: 
+
+```js
+const numbersArr = [1, 2, 3, 4, 5, 1, 2, 3, 4, 5, 1, 2, 3, 4];
+const counts = {};
+numbersArr.forEach(el => counts[el] = (counts[el] || 0) + 1)
+```
+
+In this example, the `numbersArr` has each element looped through like before. Only this time, it removes the 
+if statement and replaces it with an `or` operator. This `or` operator says if `counts[el]` is *falsy* to return the other value. 
+
+
+An value can be considered *falsy* if it is `null`, `undefined`, or otherwise can be evaluated as false. 
```

Comment:
```suggestion
An value can be considered <dfn>falsy</dfn> if it is `null`, `undefined`, or otherwise can be evaluated as false. 
```

Though I do feel like we've taught this before.

Reply:
We might have explained it in the shopping cart curriculum though it might not hurt to explain it multiple times.